pkg/kubelet/apis/config/v1beta1/defaults.go

```diff
@@ -154,6 +155,9 @@ func SetDefaults_KubeletConfiguration(obj *KubeletConfiguration) {
 	if obj.CPUCFSQuota == nil {
 		obj.CPUCFSQuota = utilpointer.BoolPtr(true)
 	}
+	if obj.CPUCFSQuotaPeriod == nil && obj.FeatureGates[string(features.CPUCFSQuotaPeriod)] {
```

Comment:
I don't know if this actually works...  We generally do not feature gate the defaulting of flags.  We generally only place the feature gate around code we don't want to run, rather than around configuration.

Reply:
This does look like a problem.  The flag should always default, it just shouldn’t have been used.  Apologies for missing in review